munkipkg

```diff
@@ -619,6 +659,92 @@ def build_distribution_pkg(build_info, options):
         raise BuildError(err)
 
 
+def notarize(build_info, options):
+    '''Use xcrun altool to notarize our package'''
+    if not (
+        'username' in build_info['notarization_info']
+        and 'password' in build_info['notarization_info']
+    ):
+        raise MunkiPkgError("notarization_info lacks username or password.")
+
+    display("Uploading package to Apple notary service", options.quiet)
+    cmd = [
+        XCRUN,
+        'altool',
+        '--notarize-app',
+        '--primary-bundle-id',
+        build_info['identifier'],
+        '--username',
+        build_info['notarization_info']['username'],
+        '--password',
+        build_info['notarization_info']['password'],
+        '--output-format',
+        'xml',
+        '--file',
+        os.path.join(build_info['build_dir'], build_info['name']),
+    ]
+    if 'asc_provider' in build_info['notarization_info']:
+        cmd.extend(
+            ['--asc-provider', build_info['notarization_info']['asc_provider']]
+        )
+    retcode, proc_stdout, proc_stderr = run_subprocess(cmd)
+    try:
+        output = readPlistFromString(proc_stdout)
+    except ExpatError:
+        print(proc_stderr, file=sys.stderr)
+        raise MunkiPkgError("Notarization failed. Unable to run xcrun altool")
+
+    if retcode:
+        for product_error in output['product-errors']:
+            print("altool: FAILURE " + product_error['message'], file=sys.stderr)
```

Comment:
same thing here: `product_error.get('message', 'UNKNOWN ERROR')`

Reply:
Implemented in ee6520c.